vnext/Shared/Networking/WinRTHttpResource.cpp

```diff
@@ -336,6 +336,22 @@ void WinRTHttpResource::SetOnData(function<void(int64_t requestId, dynamic &&res
   m_onDataDynamic = std::move(handler);
 }
 
+void WinRTHttpResource::SetOnIncrementalData(
+    function<void(int64_t requestId, string &&responseData, int64_t progress, int64_t total)> &&handler) noexcept
+/*override*/ {
+  m_onIncData = std::move(handler);
```

Comment:
If the containing method is meant to be thread-agile, should we guard against using while changing the member (e.g. via a mutex)? If the method is supposed to be called on a particular thread, can we assert that?
Same for the methods below.

Reply:
Perhaps relevant given that these callbacks appear to get used in coroutines?

Reply:
Method is not thread-agile.

Moreover, it's conceived to be called only once (by the owner: a CxxModule instance).